.github/workflows/generative-query-test.yml

```diff
@@ -0,0 +1,56 @@
+name: Generative Query Test
+
+on:
+  # push:
+  #   branches:
+  #     - 'master'
+  #     - 'release-**'
+  # pull_request:
+  #   types: [opened, synchronize, reopened, ready_for_review]
+  workflow_dispatch:
+    inputs:
+      seed:
+        description: 'Seed value.'
+        type: number
+        required: true
+        # TODO: Make this optional
+        default: 1
+      count:
+        description: 'Number of queries to generate'
+        type: number
+        required: true
+        default: 1
+
+jobs:
+  generative-query-test:
+    runs-on: ubuntu-22.04
+    timeout-minutes: 90
+    name: Generative Query Test
+    env:
+      CI: 'true'
+      MB_TEST_SEED_VALUE: ${{ inputs.seed }}
+      MB_TEST_QUERY_ITERATION_COUNT: ${{ inputs.count }}
+      MB_TEST_RUN_GENERATIVE_TEST: true
```

Comment:
Do we only want these tests to run in CI?

It feels like these parameters should have default values that will run the tests at the CLI or in the namespaces. But perhaps I'm not seeing the full picture here?